ocs_ci/helpers/dr_helpers.py

```
@@ -4453,3 +4484,265 @@ def create_cdi_cert_configmap(namespace, configmap_name="user-ca-bundle"):
+        last_exception = None
+        last_stdout = None
+        for attempt in range(1, retries + 1):
+            logger.info(
+                f"Running ODF DR validate application for DRPC '{drpc_name}' "
+                f"in namespace '{namespace}' (attempt {attempt}/{retries})"
+            )
+            try:
+                result = odf_cli_runner.run_command(cmd_args, timeout=1200)
+                last_stdout = result.stdout.decode()
+                last_exception = None
+                logger.info(f"ODF DR validate application output:\n{last_stdout}")
```

Ruff (0.16.0)

[warning] Logging statement uses f-string

(G004)